.github/workflows/Parser-Json-Release-Build.yml

```diff
@@ -0,0 +1,25 @@
+name: Parser Json Release Build
+
+on:
+  release:
+      types: [published]
+      
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v3
+        with:
+          dotnet-version: 6.0.x
+      - name: Restore dependencies
+        run: dotnet restore
+      - name: Build
+        run: dotnet build --no-restore
+      - name: Test
+        run: dotnet test --no-build --verbosity normal
+      - name: Pack Orchestrator
+        run: dotnet pack ${GITHUB_WORKSPACE}/Parsers/Prognetics.Saga.Parser.Json/Prognetics.Saga.Parser.Json.csproj -p:PackageVersion=${{ github.ref_name }} -c Release
+      - name: Publish to nuget.org
+        run: dotnet nuget push ${GITHUB_WORKSPACE}/Parsers/Prognetics.Saga.Parser.Json/bin/Debug/*.nupkg -k ${{ secrets.NUGETAPIKEY }} -s https://api.nuget.org/v3/index.json
```

Comment:
You still push nuget from the Debug directory

Reply:
Changed!